docs/FAQ.md

```diff
@@ -1,11 +1,152 @@
+[AMD SVM Spec](https://www.amd.com/system/files/TechDocs/24593.pdf)
+
+### Why TrenchBoot?
+
+- Managing each of these components individually can be confusing
+- Most DTRM systems are platform dependent
+- Most DTRM systems can only measure what was loaded into memory with it at the
+    time it was started
+- Most DTRM systems support only one attestation action
+
+## 2. How does TrenchBoot work? Trenchboot Architecture
+
+### Architecture Overview
+
+*The full architecture overview can be found at
+[trenchboot.org](https://trenchboot.org/documentation/Architecture)*
+![Overview](img/Architectural_Flow.png)
+This diagram shows a general overview of the overall architecture of a
+TrenchBoot system. In Bootstrap phase the system environment is measured and
+that information stored in the dynamic PCRs, using whatever method is
+appropriate for the given hardware. The intermediate phase analyzes the
+information collected in the bootstrap phase to decide if it wants to continue
+booting and pass control to the target OS, or if it wants to stop immediately.
+If desired, additional evidence/measurements can be gathered in this phase. Once
+ the integrity of the system is verified the TrenchBoot kernel passes off
+ control to the actual desired OS to initiate the runtime phase.
+
+### Bootstrap Phase - GRUB
+
+Typically, when attempting to establish a DRTM, TBoot is loaded in order to
+carry out TXT on Intel, or SEV is used on AMD. TrenchBoot instead utilizes its
+own system, which is a custom version of GRUB that has commands built in to
+carry out a TrenchBoot "Secure Launch". This customized grub introduces the
+slaunch and slaunch_module commands. A working example of the TrenchBoot GRUB
+can be found at
+[https://github.com/3mdeb/meta-trenchboot](
+    https://github.com/3mdeb/meta-trenchboot).
+The custom GRUB is however unfinished and while it is running on AMD, it is not
+yet functional with Intel processors.
+
+### Intermediate Phase - Linux Kernel / U-ROOT (Initramfs)
+
+The intermediate phase processes information collected in the bootstrap phase
+and converts it into normalized data. This data is eventually used to determine
+whether to boot into the target OS. The kernel and u-root initramfs
+together make up Trenchboot's "Security Engine", which is an intermediate
+mini-OS that serves to process data gathered by that bootstrap phase. During the
+intermediate phase, the kernel and initramfs work together to measure block
+devices, individual files, SMBUS/DMI information and provides unseal based,
+external device and network based attestation. Trenchboot calls this
+functionality secure launch (aka slaunch). The table in section 4 lists the
+specific repo and branch to use when building the kernel or u-root.
+
+## 3. How to use Trenchboot? Practical usage
+
+### Trenchboot GRUB TXT Config with Serial Communication
+
+*Note: ACM modules can be downloaded from
+[mirror.openxt.org](http://mirror.openxt.org/)
```

Comment:
Yes, I forgot to mention it. I was unsure if this was a temporary problem because my browser seemed to try to load it indefinitely. If you a know a better site to link to instead then that's great.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/77aaa7d4fca79f07460ba2bc9372c2e0cea0e01b..333691d2321d4a1a49e964dd41f811f4dda7daa5#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17R115